project/Doc.scala

```diff
@@ -31,7 +31,30 @@ object Scaladoc extends AutoPlugin {
       scalacOptions in Compile <++= (version, baseDirectory in ThisBuild) map scaladocOptions,
       autoAPIMappings := CliOptions.scaladocAutoAPI.get
     )) ++
-    Seq(validateDiagrams in Compile := true) ++
+    Seq(
+      validateDiagrams in Compile := true,
+      apiMappings ++= {
+        // Based on https://github.com/ThoughtWorksInc/sbt-api-mappings
+        val IvyRegex = """^.*[/\\]([\.\-_\w]+)[/\\]([\.\-_\w]+)[/\\](?:jars|bundles)[/\\]([\.\-_\w]+)\.jar$""".r
+
+        for {
+          jar <- (dependencyClasspath in Compile in doc).value.toSet ++ (dependencyClasspath in Test in doc).value
+          fullyFile = jar.data
+          url <- fullyFile.getCanonicalPath match {
+            case IvyRegex(organization, name, jarBaseFile) if jarBaseFile.startsWith(s"$name-") => {
+              val version = jarBaseFile.substring(name.length + 1, jarBaseFile.length)
+              organization match {
+                case "com.typesafe.akka" =>
```

Comment:
This ensures we at least have `apiMappings` for Akka core. It would be better to make a PR in akka/akka to configure apiURL so sbt will automatically add this for us.